libr/debug/p/debug_gdb.c

```diff
@@ -311,6 +312,21 @@ static int r_debug_gdb_reg_write(RDebug *dbg, int type, const ut8 *buf, int size
 		}
 		ut64 val = r_reg_get_value (dbg->reg, current);
 		int bytes = bits / 8;
+		if (r_config_get_i (((RCore*)dbg->corebind.core)->config, "cfg.bigendian")) {
```

Comment:
core can be null and should be moved outside the loop as bool bigEndian = dbg->corebind.cfgGet()..

if you use the corebind structure it will chk for null and will not require to link against rconfig

Reply:
Done.
Default to little-endian in case core is NULL, so the behavior is consistent with previous versions.